test/contracts/MgnToken.sol

```diff
@@ -0,0 +1,177 @@
+pragma solidity ^0.4.24;
+
+import "./TestToken.sol";
+
+// TODO: Review also if we can use the real MGN easily
+contract TokenFRT is TestToken {
```

Comment:
why not test with the real MgnToken contract ?

Reply:
We do, this files is not used anymore. I just deleted it.